rpcs3/Emu/Cell/lv2/sys_fs.cpp

```diff
@@ -77,6 +77,47 @@ lv2_fs_mount_point* lv2_fs_object::get_mp(std::string_view filename)
 	return &g_mp_sys_dev_hdd0;
 }
 
+std::size_t lv2_fs_object::count_path_items(std::string_view filename)
+{
+	std::size_t count = 0;
+
+	while (true)
```

Comment:
is it not possible to init begin in the condition and test for umax as if it was an if condition?

Reply:
its possible but takes from readability.
it looks something like `for (std::size_t begin; (begin = filename...) != unax);)

Reply:
Hmm. I always think while (true) makes you think it's some endless loop.
So idk what would make it less readable.
Anyway. It's just a suggestion

Reply:
Wouldnt it be more like:

for (std::size_t begin = filename....; begin != umax; begin = filename...)

Reply:
im not a fan of duplicating init code twice though.